.github/workflows/geth-with-protocol.yml

```diff
@@ -0,0 +1,27 @@
+name: geth-with-protocol
+on: workflow_dispatch
```

Comment:
One other thing you can consider is to add a parameter with `inputs`. Then, you could not hardcode `confluence` docker image tag, but the user could just input whatever tag they want.

Anyway, I leave it up to you.

Reply:
IMHO I think it makes sense to agree to name it `confluence` since it seems all the protocol-related docker releases have been tied up to breaking changes in the protocol (i.e.: the previous one was tagged `streamflow`)

Anyway good thing you asked me to test the GH on a fork, I actually missed a -t flag in the docker build command